docs/relay/README.md

```diff
@@ -43,7 +43,8 @@ chainlink nodes solana create --name=<node-name> --chain-id=<chain-id> --url=<ur
 | `OCR2CacheTTL`        | stale OCR2 cache deadline                                                                                                                                                                                          | 1m          |                                       |
 | `TxTimeout`           | timeout to send tx to rpc endpoint                                                                                                                                                                                 | 1m          |                                       |
 | `TxRetryTimeout`      | duration for tx to be rebroadcast to rpc, txm stops rebroadcast after timeout                                                                                                                                      | 10s          |                                       |
-| `TxConfirmTimeout`    | duration when confirming a tx signature before signature is discarded as unconfirmed                                                                                                                               | 30s         |                                       |
+| `TxConfirmTimeout`    | duration when confirming a tx signature before signature is discarded as unconfirmed                                                                                                                               | 30s         |  
+| `TxExpirationRebroadcast`       | enables or disables transaction rebroadcast if expired. Expiration check is performed every `ConfirmPollPeriod`. A transaction is considered expired if the blockhash it was sent with is 150 blocks older than the latest blockhash.                                                                                                                                                                 | `false`      | `true`, `false`                                      |
 | `SkipPreflight`       | enable or disable preflight checks when sending tx                                                                                                                                                                 | `true`      | `true`, `false`                       |
```

Comment:
Just wanted to call this out for consistency. I think the `enables or disables` is probably the more standard way to describe configs in our code base but think we should also update `SkipPreflight`'s description here if we're updating that here.
```suggestion
| `TxExpirationRebroadcast`       | enables or disables transaction rebroadcast if expired. Expiration check is performed every `ConfirmPollPeriod`. A transaction is considered expired if the blockhash it was sent with is 150 blocks older than the latest blockhash.                                                                                                                                                                 | `false`      | `true`, `false`                                      |
| `SkipPreflight`       | enables or disables preflight checks when sending tx                                                                                                                                                                 | `true`      | `true`, `false`                       |
```